### ISSUE TITLE
1.32.X Win release?

### ISSUE THREAD

**Issue by gerroon** [VSCodium/vscodium#90]
Hi

Any plans for the Win release?

thanks

**Comment by stripedpajamas** [comment]
Yes, it looks like the Windows build keeps running out of memory in the Azure Pipeline. Will troubleshoot and hopefully get the build out.

**Comment by gerroon** [comment]
Thanks

**Comment by pavanagrawal123** [comment]
@stripedpajamas I don't see a memory issue, but I see some license file not being there causing the compile to fail, which causes the rest of the tasks to fail...

**Comment by den32ds** [comment]
Still wait

**Comment by stripedpajamas** [comment]
You're right @pavanagrawal123. Seems like the inno build script is looking for LICENSE.rtf now, instead of LICENSE.txt. I can't figure out how it would ever find that or any build script that renames the existing txt file to an rtf. I suppose we could add a line that copies the existing LICENSE.txt to LICENSE.rtf to our own build script. Just feels like I'm missing something.

**Comment by den32ds** [comment]
So, what`s wrong with Win builds, if there are already 3 versions in a row?

**Comment by gerroon** [comment]
1.32.3 says

```
Update build.sh

Windows builds were not using the npm gulp script which allows for a bigger heap
```

But I see no win build, not sure if there was a fix or not.

**Comment by pavanagrawal123** [comment]
psst @stripedpajamas license change is related to this commit: https://github.com/Microsoft/vscode/commit/b81b84ff639af8de8df7c53a67874c2aee83adfa

**Comment by stripedpajamas** [comment]
Sorry for the delay, everyone. I've been without my laptop this week but will hopefully get the Windows builds up and going in the next day or so. 

The first issue I found was that the build was running out if memory and that is something I resolved. The other issue is the build is looking for a license rtf instead of txt. No info from MS about why this change was committed but it shouldn't be too tricky to work around.
